reg_tests/chgres_cube/driver.hera.sh

```diff
@@ -44,9 +44,14 @@ QUEUE="${QUEUE:-batch}"
 # and baseline data for each test.
 #-----------------------------------------------------------------------------
 
+#export UPDATE_BASELINE="FALSE"
+export UPDATE_BASELINE="TRUE"
+
+source ../get_hash.sh
+
```

Comment:
Should this only be sourced if UPDATE_BASELINE="TRUE"?

Reply:
Do we manually set UPDATE_BASELINE="TRUE" and rerun the RTs when we know we want to replace the baselines?

Reply:
> Should this only be sourced if UPDATE_BASELINE="TRUE"?

Good suggestion. I will make that change.

Reply:
> Do we manually set UPDATE_BASELINE="TRUE" and rerun the RTs when we know we want to replace the baselines?

Yes. After a merge that changes results, the corresponding scripts would be run with UPDATE_BASELINE="TRUE" on all machines. This should only have to be done once after a merge.

Reply:
Sounds good!